### PR TITLE
2016 mn tests for document api

### PR DESCRIPTION
**Pull request by MagdaN**

### CODE REVIEW

File: euth/documents/api.py [hunk 1]
```diff
@@ -10,5 +10,5 @@ class DocumentViewSet(viewsets.ModelViewSet):
     queryset = Document.objects.all()
     serializer_class = DocumentSerializer
     permission_classes = (
-        IsModerator,
+        permissions.IsAuthenticatedOrReadOnly, IsModerator,
```
Comment (slomo) [inline euth/documents/api.py]: This is fixing an exception in the moderator exception if the user isn't logged in? Can this be fixed in the `isModerator` permission?

Comment (MagdaN) [inline euth/documents/api.py]: Its done as in ratings and comments.

Comment (slomo) [inline euth/documents/api.py]: But why is it is `AuthenticatedOrReadOnly`?

Comment (MagdaN) [inline euth/documents/api.py]: OK, dann nehmen wir http://www.django-rest-framework.org/api-guide/permissions/#isauthenticated

**Pull request merged**